hack/.ci/run-e2e-gke.sh

```diff
@@ -60,6 +60,46 @@ kubectl_create -f - <<EOF
     - patch
 EOF
 
+# FIXME: remove the workaround once https://github.com/scylladb/scylla-operator/issues/749 is done
+kubectl_create -n default -f - <<EOF
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+    name: sysctl
+  spec:
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: sysctl
+    template:
+      metadata:
+        labels:
+          app.kubernetes.io/name: sysctl
+      spec:
+        containers:
+        - name: sysctl
+          securityContext:
+            privileged: true
+          image: "${SO_IMAGE}"
+          imagePullPolicy: IfNotPresent
+          command:
+          - /usr/bin/bash
+          - -euExo
+          - pipefail
+          - -O
+          - inherit_errexit
+          - -c
+          args:
+          - |
+            sysctl fs.aio-max-nr=0xffffffff
+
+            sleep infinity &
+            wait
+        nodeSelector:
+          kubernetes.io/os: linux
+          pool: workers
```

Comment:
I think `"scylla.scylladb.com/node-type": "scylla"` would be more fitting and stable

Reply:
I did that initially but changed to `pool: workers` to align with existing manifests: https://github.com/scylladb/scylla-operator/blob/25735c38e04ee9deb0fa5f471f2ce7951aff5b16/hack/.ci/manifests/namespaces/local-csi-driver/50_daemonset.yaml#L19. Does `pool: workers` have a different meaning, i.e. is worker pool potentially a superset of scylla node pool? Anyway `"scylla.scylladb.com/node-type": "scylla"` is neater so I'll change it.

Reply:
thanks. not really, but `scylla.scylladb.com/node-type` is more standard and we use it in our docs. `pool: workers` for probably for some convenience and could go away.